sdk/core/azure-core/azure/core/polling/base_polling.py

```diff
@@ -198,15 +198,15 @@ class OperationResourcePolling(LongRunningOperation):
     """
 
     def __init__(
-        self, operation_location_header="operation-location", **kwargs
+        self, operation_location_header="operation-location", *, lro_options={}
```

Comment:
Never default to a dict, ever :)

Reply:
This is our original code:

kwargs.pop("lro_options", {}) or {}

Reply:
Updated.